### ISSUE TITLE
Implement tags and description resources

### ISSUE THREAD

**Issue by Pedro270707** [Wilyicaro/Legacy-Minecraft#25]
Look at the following in LCE UI mod (I don't know how to explain what I want to explain):
[`resources/assets/lceui/tags/descriptions`](https://github.com/Pedro270707/LCE-UI/tree/1.20/src/main/resources/assets/lceui/tags/descriptions)
[`resources/assets/minecraft/descriptions`](https://github.com/Pedro270707/LCE-UI/tree/1.20/src/main/resources/assets/minecraft/descriptions)
[`TagResourceLoader.java`](https://github.com/Pedro270707/LCE-UI/blob/1.20/src/main/java/net/kyrptonaught/lceui/resourceloaders/TagResourceLoader.java)
[`DescriptionResourceLoader.java`](https://github.com/Pedro270707/LCE-UI/blob/1.20/src/main/java/net/kyrptonaught/lceui/whatsThis/resourceloaders/DescriptionResourceLoader.java)

Basically, strings like `block.minecraft.stained_glass.description` won't work because there's no `minecraft:stained_glass` block, only `minecraft:red_stained_glass`, `minecraft:orange_stained_glass` etc. This goes for any block groups (wool, spawn eggs etc.)

**Comment by Wilyicaro** [comment]
I'm not going to implement this at all, as it's not efficient just to get a tip/description translation, I'm going to add tip overrides for these cases, specifying tags or list of blocks with the same description
